lisa/tests/base.py

```diff
@@ -469,8 +469,17 @@ class RTATestBundleMeta(abc.ABCMeta):
     default set of events is always enough to run all defined methods, without
     duplicating that information.
 
-    .. note:: An existing ``ftrace_conf`` attribute is used, with extra
-        detected events merged-in.
+    The means that the ``ftrace_conf`` attribute is typically built by merging
```

Comment:
```suggestion
    The ``ftrace_conf`` attribute is typically built by merging
```
Reads slightly better IMO